prime-router/docs/onboarding-users/senders.md

```diff
@@ -63,6 +63,30 @@ The schema provides additional transforms required so that the senders data can
 
 Note: When creating a transform, please reference [Changing/Updating Sender/Receiver Transforms](../getting-started/standard-operating-procedures/changing-transforms.md) for guidance.
 
+### Authenticating to ReportStream’s REST API
+**Note: This is legacy authentication and should not be used for onboarding new users**
+
+Shared secret key authorization
```

Comment:
This is mis-labled. Th steps below are to generate a public/private keypair for token auth.